scripts/sampinctrl.py

```diff
@@ -0,0 +1,268 @@
+# Copyright (c) 2021 Teslabs Engineering S.L.
+# Copyright (c) 2022 Gerson Fernando Budke
+# SPDX-License-Identifier: Apache 2.0
+
+"""
+Utility to autogenerate pinctrl definitions.
+
+Usage::
+    python3 sampinctrl.py [-i /path/to/configs] [-o /path/to/include]
+
+Copyright (c) 2021 Teslabs Engineering S.L.
+Copyright (c) 2022 Gerson Fernando Budke
+SPDX-License-Identifier: Apache 2.0
```

Comment:
nit: you can remove this as it's already at the top of this file

Reply:
Done!